### ISSUE TITLE
Execute-ProcessAsUser still not working properly in new v3.6.4 release

### ISSUE THREAD

**Issue by appPackager** [PSAppDeployToolkit/PSAppDeployToolkit#38]
A double-quote is missing from the end of the PSAppDeployToolkit-ExecuteAsUser.vbs which results in error: "Unterminated string constant". The fix is to add a double-quote to the end of the vbscript (i.e. ; Exit $LastExitCode }"). My PSAppDeployToolkit code is as follows:
```
Execute-ProcessAsUser -Path "$PSHOME\powershell.exe" -Parameters "-Command & { & `"C:\Temp\Script.ps1`"; Exit `$LastExitCode }" -Wait
```

The resulting PSAppDeployToolkit-ExecuteAsUser.vbs is as follows:
```
strCommand = chr(34) & "C:\windows\System32\WindowsPowerShell\v1.0\powershell.exe" & chr(34) & " -Command & { & " & chr(34) & "C:\Temp\Script.ps1" & chr(34) & "; Exit $LastExitCode }
set oWShell = CreateObject("WScript.Shell")
intReturn = oWShell.Run(strCommand, 0, true)
WScript.Quit intReturn
```

**Comment by mmashwani** [comment]
Well now I'm confused. I removed a double quote because of this other ticket last time around:
https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/issues/22

However, you're saying that it needs to be put back. Can you confirm?

Did you change your script to the below?
```
[string]$executeProcessAsUserParametersVBS = 'chr(34) & ' + "`"$($Path)`"" + ' & chr(34) & ' + '" ' + ($Parameters -replace '"', "`" & chr(34) & `"" -replace ' & chr\(34\) & "$','') + '"'
```

**Comment by appPackager** [comment]
I think the root cause of the issue reported in #22 was the help example was wrong as reported in #28.  The fix that was applied as a result of #22 was unnecessary. 

I replaced the line within Execute-ProcessAsUser as you suggested and Execute-ProcessAsUser now generates a good .vbs resulting in a successful execution.

**Comment by mmashwani** [comment]
OK, makes sense. I've reverted the change made in the last release so since the example was already fixed, both issues should now be fixed.
